.github/workflows/test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - run: git config --global user.email CI@GITHUB
 
       # Run all tests
-      - run: poetry install
+      - run: poetry install --with dev
```

Comment:
Why this change?

Reply:
The CI failures for previous commit are weird. I don't get those testing locally. I was wondering if maybe pre-commit or pyyaml were too old. Or maybe if a more modern poetry isn't installing dev dependencies, as older poetry did. They had quite a big change in [dependency groups management](https://python-poetry.org/docs/managing-dependencies#dependency-groups).